lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -136,13 +169,16 @@
 def db_exists(cursor, db):
     res = 0
     for each_db in db:
-        res += cursor.execute("SHOW DATABASES LIKE %s", (each_db.strip().replace("_", r"\_"),))
+        res += cursor.execute("SHOW DATABASES LIKE %s", (each_db.replace("_", r"\_"),))
```

Comment:
Good point for handling the `_` char, but have you tested the `-` char ?
I know some devs use it 🙄 in the db name, as long as they quote the db name.

Reply:
@bmalynovytch: Yes, it does work for `-` char as well.

Ansible-playbook:
```
$ cat only_create.yml 
---
- name: Sample
  gather_facts: no
  hosts: localhost
  tasks:
  - name: Create multiple databases
    mysql_db:
      name:
        - "abcd-pratik"
      #state: present
      state: absent
      login_host: 172.17.0.2
      login_user: monty
      login_password: pw
    register: result
  - debug:
      msg: "{{ result }}"
```

Ansible Run:
```
$ ansible-playbook only_create.yml 
 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'


PLAY [Sample] *****************************************************************************************************************************************************************************************************

TASK [Create multiple databases] **********************************************************************************************************************************************************************************
changed: [localhost]

TASK [debug] ******************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": {
        "changed": true, 
        "db": "abcd-pratik", 
        "db_list": [
            "abcd-pratik"
        ], 
        "failed": false
    }
}

PLAY RECAP ********************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

Reply:
Marvelous ! 👏